.github/workflows/cut-release-branch.yml

```diff
@@ -34,7 +34,8 @@ on:
   workflow_call:
     inputs:
       version_to_bump_main:
-        required: true
+        description: "DEPRECATED: The version to bump main to.  This is no longer used."
```

Comment:
You should create a new ticket to clean this up and remove it from all the calling workflows in the repos.

Reply:
https://github.com/dbt-labs/actions/issues/115